docs/posts/introduction-to-ballerina-0.970.md

```diff
@@ -0,0 +1,220 @@
+
+## Messaging endpoints
+Messaging connectors enable the services and programs written in Ballerina to communicate with messaging backends like Ballerina Message Broker and ActiveMQ. The messaging connector API provides features like transactions, message headers, properties, and different acknowledgement mode support to address the common requirements of a modern integration engineer when integrating with a messaging system.
+
+## WebSub
+Implementation of the WebSub recommendation that facilitates push-based content delivery/notification mechanism between publishers and subscribers.
+
+# Standard Library 
+The Ballerina standard library provides a set of commonly used functionalities. The following packages are available as a part of the standard library:
+
+## ballerina/auth
+Provides an interface for looking up user data for authentication and authorization purposes. Also it contains a sample implementation that uses a Ballerina configuration file as user registry.
+
+## ballerina/cache
+Provides a configurable in-memory caching solution that supports both time-based eviction and size-based eviction.
+
+## ballerina/config
+Provides a configuration lookup and resolve mechanism, with the option for securing configurations, and an API for reading these configurations.
+
+## ballerina/crypto
+Provides a set of functions for some of the commonly used hashing algorithms.
+
+## ballerina/file
+Provides a directory listener that can be used to listen to directory events.
+
+## ballerina/io
+Provides an asynchronous I/O framework to source/sink that reads/writes as bytes, characters, and records.
+
+## ballerina/log
+Provides an API for logging.
+
+## ballerina/sql
+Provides the common types and constants used across all the database client endpoints.
+
+
+## ballerina/math
+Provides a set of functions for performing commonly used mathematical calculations and operations.
+
+## ballerina/reflect
+Provides utility functions for reading annotations and deep equality checks.
+
+## ballerina/runtime
+Provides utility functions and records related to the runtime.
+
+## ballerina/system
+Provides a set of functions for retrieving details related to the system.
+
+## ballerina/task
+Provides an API for managing task timers and task appointments.
+
+## ballerina/time
+Provides a set of functions for handling, parsing and formatting date and time.
+
+
+# IDEs and Language Server 
+
+## Language Server 
+Ballerina Language Server provides the code intelligence for Ballerina programming. Ballerina Language Server can be integrated with any Language Server Protocol (LSP) supported development tool to provide consistent code intelligence throughout. 
+
+## VSCode Plugin
+The Ballerina VSCode plugin includes the following features: - Syntax highlighting - Intellisense for Ballerina language via Ballerina Language Server - Diagramming (view Ballerina programs graphically) - Debugging
```

Comment:
Shall we make these bullet points? Please do the same in the following subsection as well.